packages/block-library/src/group/style.scss

```diff
@@ -0,0 +1,4 @@
+.wp-block-group {
+	background-color: var(--wp--background-color);
+	color: var(--wp--text-color);
```

Comment:
@nosolosw Is this the correct variable syntax for global styles? Will this make the `theme.json` config automatically work?

Reply:
As it is currently in master, it'd be the other way around: `--wp--color--text`, `--wp--color--background` (note also the double hyphen).

Reply:
I guess this is optimized for the `theme.json` file hierarchy. we need to make a decision about whether we keep it or try to simplify it because that same hierarchy ends up being in the `style` attribute of each block.

Reply:
I matached these.